include/enum_quda_fortran.h

```diff
@@ -17,11 +17,6 @@
 #define QUDA_ERROR 1
 #define QUDA_ERROR_UNINITIALIZED 2
 
-#define QUDA_MEMORY_DEVICE 0
```

Comment:
Just confirming this full delete was intentional, and there's no need to expose these `enum` values via the fortran interface?

Reply:
Exactly.